src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -561,6 +545,29 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         indexRegistry.deregisterOperator(operatorId, quorumNumbers);
     }
 
+    /**
+     * @notice update operator stake for specified quorumsToUpdate, and deregister if necessary
+     * does nothing if operator is not registered for any quorums.
+     */
+    function _updateOperator(
+        address operator,
+        OperatorInfo memory operatorInfo,
+        bytes memory quorumsToUpdate
+    ) internal {
+        if (operatorInfo.status != OperatorStatus.REGISTERED) {
+            return;
+        }
+        bytes32 operatorId = operatorInfo.operatorId;
+        uint192 quorumsToRemove = stakeRegistry.updateOperatorStake(operator, operatorId, quorumsToUpdate);
+
+        if (!quorumsToRemove.isEmpty()) {
```

Comment:
cant we check quorumsToUpdate.length == 0

Reply:
Some quorums might fall below minimum stake and are marked to be removed in the `quorumToRemove` bitmap from the `stakeRegistry.updateOperatorStake` call so we check the value it returns